problems/oldestPerson.js

```diff
@@ -11,8 +11,17 @@
  * ex: oldestPerson({ 'Marcus': 29, 'Julia': 29, 'Kevin': 34 })
  * returns: 'Kevin'
  */
-function oldestPerson() {
+function oldestPerson(object) {
+  let arrOldestPerson = [];
 
+  for (let key in object) {
+    arrOldestPerson.push([key, object[key]]);
+  }
+  arrOldestPerson.sort((a, b) => {
+    return b[1] - a[1];
+  });
+  return arrOldestPerson[0][0];
```

Comment:
That was a cool way of solving this problem.

Alternative would be to keep track of two variables, name and age, and update them as you iterate through the object.